test/issuehunter.js

```diff
@@ -1284,7 +1284,7 @@ contract('Issuehunter', function (accounts) {
         }).then(function () {
           return verifyPatch(issueId, author, ref, patchVerifier)
         }).then(function () {
-          return increaseTime(60 * 60 * 24 * 8)
+          return increaseTime((60 * 60 * 24 * 7) - 1)
```

Comment:
This should be `((60 * 60 * 24 * 8) - x)`, where `x` is a small number of seconds, ideally between 2 and 5.

You should do the same in the `right before the pre-reward period end` context, and mark https://github.com/issuehunter/issuehunter/issues/15 as complete.

Reply:
@potomak isn't 7 by default?

Reply:
It's 1 day (pre-reward period) + 7 days (reward period).